policy/modules/services/xserver.te

```diff
@@ -883,7 +883,7 @@ optional_policy(`
 ')
 
 optional_policy(`
-	resmgr_stream_connect(xdm_t)
+	resmgr_stream_connect(xserver_t)
```

Comment:
I'm ok with this being in the PR, but it should be a separate commit since it is a correction.

Reply:
I think this is still relevant.